openmetadata-ui/src/main/resources/ui/src/components/DatasetDetails/DatasetDetails.component.tsx

```diff
@@ -633,6 +633,7 @@ const DatasetDetails: React.FC<DatasetDetailsProps> = ({
                     constraint: col.constraint as string,
                     colName: col.name,
                     colType: col.dataTypeDisplay as string,
+                    dataType: col.dataType as string,
```

Comment:
@ShaileshParmar11 , Why are we changing this from `dataTypeDisplay` to `dataType`?

Reply:
Hi @Sachin-chaurasiya, i have't changed anything. i have added dataType as i need the dataType of column to identify whether we support the test for particular column.

Reply:
@ShaileshParmar11, My bad.

Reply:
No issue @Sachin-chaurasiya, can we resolve this?